### ISSUE TITLE
2021-9-26

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#614]
461Set LWF environment on local; done; successfully run LWF at local; (2021-9-22)
462Replace beforeTrain with before_train; done; replace afterTrain with after_train; done; (2021-9-22)
463Replace network with Network; done; replace Incremental_learning with incremental_learning; done; (2021-9-24)
464Replace LwFmodel with LwfModel; done; replace numclass with num_class; done; (2021-9-24)
465Replace batchsize with batch_size; done; replace _compute_loss with compute_loss; done; (2021-9-24)
466Warning: Comparison with None performed with equality operators ; you should better use : if oim.sent is not None: done; (2021-9-24)
467Replace Image_transform with image_transform; done; replace _test with test_model; done; (2021-9-24)
468What is binary_cross_entropy_with_logits? Function that measures Binary Cross Entropy between target and output logits; (2021-9-24)
469model.eval() is a kind of switch for some specific layers/parts of the model that behave differently during training and inference (evaluating) time. For example, Dropouts Layers, BatchNorm Layers etc. You need to turn off them during model evaluation, and .eval() will do it for you. In addition, the common practice for evaluating/validation is using torch.no_grad() in pair with model.eval() to turn off gradients computation; (2021-9-24)
